src/Symfony/Bundle/SecurityBundle/Tests/Functional/Bundle/CsrfFormLoginBundle/Controller/LoginController.php

```diff
@@ -21,9 +21,9 @@ public function loginAction()
     {
         $form = $this->container->get('form.factory')->create('Symfony\Bundle\SecurityBundle\Tests\Functional\Bundle\CsrfFormLoginBundle\Form\UserLoginType');
 
-        return $this->container->get('templating')->renderResponse('CsrfFormLoginBundle:Login:login.html.twig', array(
+        return $this->container->get('templating')->renderResponse('CsrfFormLoginBundle:Login:login.html.twig', [
```

Comment:
I know @fabpot wanted to prevent these kind of changes as they may cause merge conflicts, and it looks inconsistent.